packages/hardhat/hardhat.config.js

```diff
@@ -15,6 +15,9 @@ task("accounts", "Prints the list of accounts", async (taskArgs, hre) => {
 });
 
 const rinkebyURL = `https://rinkeby.infura.io/v3/${process.env.INFURA_ID}`;
+// const mumbaiURL = `https://polygon-mumbai.infura.io/v3/${process.env.INFURA_ID}`;
+const mumbaiURL = `https://polygon-mumbai.g.alchemy.com/v2/${process.env.ALCHEMY_ID}`;
+const maticURL = `https://polygon-mainnet.infura.io/v3/${process.env.INFURA_ID}`;
```

Comment:
Just curious why mumbai is alchemy

Reply:
I was having issues with infura at the time, might've been temporary. Alchemy was just my first choice for a backup.